spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -639,14 +766,15 @@ function cancelChannelUpgrade(
     abortTransactionUnless(verifyChannelUpgradeError(connection, proofHeight, proofUpgradeError, currentChannel.counterpartyPortIdentifier, currentChannel.counterpartyChannelIdentifier, errorReceipt))
 
     // cancel upgrade
-    // and restore original conneciton
+    // and restore original channel
     // delete unnecessary state
-    originalChannel = privateStore.get(channelRestorePath(portIdentifier, channelIdentifier))
+    currentChannel.state = OPEN
+    currentChannel.flushStatus = NOTINFLUSH
     provableStore.set(channelPath(portIdentifier, channelIdentifier), originalChannel)
```

Comment:
should we fast forward our upgrade sequence upon canceling an upgrade if the error receipt is > our upgrade sequence?